test/test_citusdb_baseRI.yml

```diff
@@ -0,0 +1,165 @@
+---
+- hosts: cdbc01
+  name: Citus RealTime Analytics Test
+  tasks:
+    - name: Download users data (31.7MB)
+      get_url:
+        url: https://examples.citusdata.com/tutorial/users.csv
+        dest: /tmp/users.csv
+        mode: 0755
+
+    - name: Download events data (74.7MB)
+      get_url:
+        url: https://examples.citusdata.com/tutorial/events.csv
+        dest: /tmp/events.csv
+        mode: 0755
+
+    - name: "create table users"
+      become: yes
+      become_user: postgres
+      become_method: sudo
+      command: >
+        psql -c "create table github_users(user_id bigint,url text,login text,avatar_url text,
+        gravatar_id text,display_login text);"
+      register: create_table_users_sql_results
+
+    - name: create table events
+      become: yes
+      become_user: postgres
+      become_method: sudo
+      command: >
+        psql -c "create table github_events(event_id bigint, event_type text, event_public boolean, 
+        repo_id bigint, payload jsonb,repo jsonb,user_id bigint,org jsonb, created_at timestamp);"
+      register:  create_table_events_sql_results
+
+    - name: create event index
+      become: yes
+      become_user: postgres
+      become_metho: sudo
+      command: psql -c "CREATE INDEX event_type_index ON github_events (event_type);"
+     
+    - name: create payload index
+      become: yes
+      become_user: postgres
+      become_metho: sudo
+      command: psql -c "CREATE INDEX payload_index ON github_events USING GIN (payload jsonb_path_ops);"
+
+    - name: create distributed table users
+      become: yes
+      become_user: postgres
+      become_metho: sudo
+      command: psql -c "SELECT create_distributed_table('github_users', 'user_id');"
+      register: create_dist_users_sql_results
+
+    - name: create distributed table events
+      become: yes
+      become_user: postgres
+      become_metho: sudo
+      command: psql -c "SELECT create_distributed_table('github_events', 'user_id');"
+      register: create_dist_events_sql_results
+
+    - name: import user data
+      become: yes
+      become_user: postgres
+      become_method: sudo
+      command: >
+        psql -c "\copy github_users from '/tmp/users.csv' with csv;"
+
+    - name: import events data
+      become: yes
+      become_user: postgres
+      become_method: sudo
+      command: >
+        psql -c "\copy github_events from '/tmp/events.csv' with csv;"
+
+### Verification Queries
+    - name: select counts on user data
+      become: yes
+      become_user: postgres
+      become_method: sudo
+      command: >
+        psql -c "SELECT count(*) FROM github_users"
+      register: select_counts_userdata_sql_results
+
+    - name: printout select counts on users results
+      debug:
+        msg: "{{ item[2] }}"
+      with_nested: "{{ select_counts_userdata_sql_results.stdout_lines }}"
```

Comment:
@lel99999 You could potentially use the `assert` module instead of (or in addition to) the `debug` module, which would return an error if the count doesn't match the correct value: http://docs.ansible.com/ansible/latest/assert_module.html